libs/utils/utils.js

```diff
@@ -24,6 +24,23 @@ export function getMetadata(name) {
   return meta && meta.content;
 }
 
+export default function createTag(tag, attributes, html) {
```

Comment:
There's also a `createTag` here:
https://github.com/adobecom/milo/blob/main/tools/translation/utils.js#L20

We should remove that one and use yours - can be done after this PR